### ISSUE TITLE
Church west wall contamination with bird dropping

### ISSUE THREAD

**Issue by PaulTran54** [hungple/mht-building#2]
Information

Name of Inspector: Paul Tran
Building: Church
Room or Area: Western side 
Inspect Date: 1/14/2018

Cleanliness & Neatness

Is clean and neat? (yes/no/na): Contaminated by bird dropping

Assessment/Observation/Comment:

Bio hazardous bird drop potentially cause many inhale diseases when it become airborne. Bird drop from 8 unused fans and new gas pipe. 

**Physical Structure**

Any damage or brokenness? (yes/no/na): Contaminating the pipe, fans, church wall, the Saint statues and the concrete foundation.

**Assessment/Observation/Comment:**

Suggest to remove the eight unused fans, and clean and cover the gas pipe with the galvanized sheet metal (135 degree from pipe to the wall)

**Safety Concerns**

Electrical, Mechanical, Ergonomic (yes/no/na): yes, (In a long run the gas pipe line will be corrosed and leaked. Potentially cause gas leak and fire.

**Comment by hungple** [comment]
_Electrical, Mechanical, Ergonomic (yes/no/na): yes, (In a long run the gas pipe line will be corrosed and leaked. Potentially cause gas leak and fire._
Wow.  Thanks Paul.  I have never thought about this.

**Comment by hungple** [comment]
This is how I would do to stop the birds resting on top of the gas pipe.
![birdpanel](https://user-images.githubusercontent.com/25112201/35179611-b97f6f82-fd52-11e7-80df-afb988f7666e.png)
blue: gas pipe
brown: wood panel to block the birds resting on top of the pipe.

**Comment by PaulTran54** [comment]
We may need a thin sheet of aluminum on top of wood and raise the angle to create slippery.

**Comment by hungple** [comment]
Agree

**Comment by PaulTran54** [comment]
Price from Home Depot and Harbor Freight.
Galvanized Roll Valley Flashing:  6 x (25'x12") = $217.32+ $20.10 tax = $237.42 (HomeDepot)
Blind Rivet set 3/16"x1/2" 3 boxes = $15 + 1.38 tax = $16.38 (Harbor Freight)
Total supplies cost = $ 254 
Water compressed jet rent or buy, water hose ? Use to clean dirty pipe.

![20180505_122246](https://user-images.githubusercontent.com/35464534/39666877-9e97a788-505f-11e8-853b-e4460b226222.jpg)

**Comment by hungple** [comment]
Yes.  This is great.  This is way better than the spikes.  Birds can build their nests on the spikes.  What is the total cost?

**Comment by PaulTran54** [comment]
Will discuss who buy, build and install in the meeting. I do not have truck, tools, and healthy enough to install them.

**Comment by hungple** [comment]
Action:
1. Submit the proposal above to diocese.  We can submit along with #3

**Comment by PaulTran54** [comment]
In order to build the sheet panels we need dimension of the pipe's supporter to the wall for each section.
But I do not know how to do it. Reason: the pipes are too high to reach. Any comment are welcome.

**Comment by Thien2018** [comment]
Can we build them in whatever length the flashing comes in and do the cutting and bracketing on the spot during the installation, since it’s sheet metal?

On May 5, 2018, at 12:45 PM, PaulTran54 <notifications@github.com<mailto:notifications@github.com>> wrote:


In order to build the sheet panels we need dimension of the pipe's supporter to the wall for each section.
But I do not know how to do it. Reason: the pipes are too high to reach. Any comment are welcome.

**Comment by hungple** [comment]
Can we build it ourself?  If we need to hire somebody to do so then it won't be easy to do.

**Comment by PaulTran54** [comment]
We can built in on the ground and ask someone to install it. The hard part is the installation. We must ask Father Vincent for opinion, It may cover skill and injury insurance for the worker. Best condition is 8 fan covers to be removed and the gas pipe is cleaned.

**Comment by hungple** [comment]
Can we rent this https://compactpowerrents.com/rental-equipment/aerial-equipment/19-ft-scissor-lift/ and install it ourself?  Of course we need to wait until they have final decision about removing the fan covers.

**Comment by PaulTran54** [comment]
Pipe cover installation procedure: 
1. 25 feet galvanized sheet has 12” width, we cut off 2 feet to make a 12 of ( 2”X12”) brackets. 
2. 23 feet left we will bent 37 degree angle with 6 ½ in. x 5 ½ in, the shorter side will face the wall. 
3.  We need three long nylon cables (lenght =  twice the height of the pipe to ground ), so that we able to pull the sheet metal to near the pipe and make the cutting marks at each pipe supporters. 
4. Lower the sheet metal to ground. To install the brackets and make a necessary cuts. The bracket location ideally 2 feet apart, 1 feet away from the end and/or the pipe supporter. 
5. Lift up the sheet metal and dropping to the location between wall and pipe. Fastening the bracket to the gas pipe with 3/16in x 1/2in rivet.  ( After cutting the sheet metal may need to be support by by wood studs  or pcp pipes otherwise it will be bent when we pull it up.)

**Comment by PaulTran54** [comment]
We may check the installation area first. What is space require for this lifter operate?

**Comment by hungple** [comment]
What tool do we need to bent 37 degree angle?
